python/orca/example/data/spark_pandas.py

```diff
@@ -51,4 +51,13 @@ def process_feature(df, awake_begin=6, awake_end=23):
     trans_data_shard = data_shard.transform_shard(process_feature)
     data2 = trans_data_shard.collect()
 
+    from bigdl.orca.data.transform import *
+    scale = MinMaxScaler(inputCol=["hours", "awake"], outputCol="x_scaled")
+    data_shard = scale.fit_transform(trans_data_shard)
+    data_shard.rdd.count()
```

Comment:
(1) change to `data_shard.compute()`
(2) why do you need to compute it here anyway?

Reply:
I call `count` here as I want to ensure the backend rdd is actually computed to ensure the new added function can succesfully run. I think by default `compute` is called during `__init__` in eager fashion, after we apply some transformation to the shards, we need actively call `compute` to trigger the action?

Reply:
> I call `count` here as I want to ensure the backend rdd is actually computed to ensure the new added function can succesfully run. I think by default `compute` is called during `__init__` in eager fashion, after we apply some transformation to the shards, we need actively call `compute` to trigger the action?

There is no need to `compute` if you are creating a new `SparkXShards`

Reply:
Make sense, will remove the `compute`